src/scripts/OSFramework/OSUI/Pattern/Tooltip/Tooltip.ts

```diff
@@ -684,8 +682,7 @@ namespace OSFramework.OSUI.Patterns.Tooltip {
 				Helper.A11Y.AriaHiddenFalse(this._tooltipBalloonWrapperElem);
 			}
 
-			// Trigger platform's _platformEventInitializedCallback client Action
-			Helper.AsyncInvocation(this._platformEventInitializedCallback, this.widgetId);
+			this.finishBuild();
```

Comment:
This is really really strange to be invoked here...

Reply:
Already mentioned why above :)